### ISSUE TITLE
test_omnisci_array_operators[countOf, in, not_in] crashes omniscidb server when CUDA enabled

### ISSUE THREAD

**Issue by pearu** [xnd-project/rbc#107]
```
NvidiaKernel.cpp:101 Check failed: cuLinkComplete(link_state, &cubin, &cubinSize) == CUDA_SUCCESS (999 == 0)
```

**Comment by pearu** [comment]
With https://github.com/omnisci/omniscidb-internal/pull/4714 we have:
```
2020-08-10T16:32:32.723277 F 27188 4 NvidiaKernel.cpp:132 Check failed: cuLinkComplete(link_state, &cubin, &cubinSize) == CUDA_SUCCESS (999 == 0) error   : Undefined reference to 'calloc' in 'Input 2'
Aborted (core dumped)
```

**Comment by pearu** [comment]
Resolved via #209
